File: app/src/main/java/com/alvinhkh/buseta/nlb/model/NlbRouteStop.java
package com.alvinhkh.buseta.nlb.model;

import com.google.gson.annotations.SerializedName;

public class NlbRouteStop {

    @SerializedName("route_id")
    public String route_id;

    @SerializedName("stop_sequence")
    public String stop_sequence;

    @SerializedName("stop_id")
    public String stop_id;

    @SerializedName("fare")
    public String fare;

    @SerializedName("fare_holiday")
    public String fare_holiday;

    @SerializedName("some_departure_observe_only")
    public Integer some_departure_observe_only;

    public String toString() {
        return "NlbRouteStop{route_id=" + route_id + ", stop_sequence=" + stop_sequence +
                ", stop_id=" + stop_id + ", fare=" + fare + ", fare_holiday=" + fare_holiday +
                ", some_departure_observe_only=" + some_departure_observe_only + "}";
    }

}

File: app/src/main/java/com/alvinhkh/buseta/nlb/model/NlbRouteStop.kt
package com.alvinhkh.buseta.nlb.model

import com.google.gson.annotations.SerializedName

data class NlbRouteStop(
        @SerializedName("route_id")
        var route_id: String = "",

        @SerializedName("stop_sequence")
        var stop_sequence: String = "",

        @SerializedName("stop_id")
        var stop_id: String = "",

        @SerializedName("fare")
        var fare: String = "",

        @SerializedName("fare_holiday")
        var fare_holiday: String = "",

        @SerializedName("some_departure_observe_only")
        var some_departure_observe_only: Int = 0
)